package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "ember-deploy-ddb",
-  "version": "0.0.3",
-  "description": "ember-cli-deploy index-adapter for AWS DynamoDB",
+  "name": "ember-cli-deploy-dynamodb",
```

Comment:
I changed the name of the package following the guidelines here: http://ember-cli.com/ember-cli-deploy/docs/v0.5.x/writing-a-plugin/.